src/main/java/com/project/appointments/controller/DoctorController.java

```diff
@@ -0,0 +1,54 @@
+package com.project.appointments.controller;
+
+import com.project.appointments.model.entity.Doctor;
+import com.project.appointments.service.DoctorService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1/doctor")
```

Comment:
btw, according to name conventions should use multiple names in endpoint "doctors" 
in other places should follow the same rules